packages/core/test/crypto.spec.ts

```diff
@@ -136,98 +132,51 @@ describe("Crypto", () => {
     });
   });
 
-  describe("encrypt", () => {
```

Comment:
So `encrypt/decrypt` can go completely? @IljaDaderko are you aware of anywhere this may be relied on atm?

Reply:
Noup, crypto is used within the engine that only accesses `encode/decode`